docs/opcodes/F0.mdx

```diff
@@ -7,7 +7,7 @@ group: System operations
 
 ## Notes
 
-Creates a new contract. Enters a new sub [context](/about) of the calculated destination address and executes the provided initialisation code, then resumes the current context.
+Creates a new contract using the initialisation code provided as [calldata](/about). Enters a new sub [context](/about) of the calculated destination address and executes the initialisation code, then resumes the current context.
```

Comment:
I would disagree here, this is not calldata at all, it's the actual code to the constructor. Which is explained in the input section.

Reply:
Good point, but I still feel like an explanation is needed. How about:

> Creates a new contract using the initialisation code provided in the data field of the transaction